client/src/js/viruses/components/Detail/RemoveVirus.js

```diff
@@ -25,7 +25,7 @@ class RemoveVirus extends React.Component {
                     <Button
                         bsStyle="danger"
                         icon="checkmark"
-                        onClick={this.handleConfirm}
+                        onClick={this.handleConfirm.bind(this)}
```

Comment:
Make ``handleConfirm`` an arrow function in the class definition. It will be bound to the component automatically.

```javascript
handleConfirm = () => {...}
```